scio-smb/src/main/scala/org/apache/beam/sdk/extensions/smb/ParquetTypeSortedBucketIO.scala

```diff
@@ -35,13 +36,25 @@ object ParquetTypeSortedBucketIO {
 
   def read[T: Coder: ParquetType](tupleTag: TupleTag[T]): Read[T] = Read(tupleTag)
 
-  def write[K: ClassTag, T: ClassTag: Coder: ParquetType](keyField: String): Write[K, T] =
-    Write(keyField)
+  def write[K: ClassTag, T: ClassTag: Coder: ParquetType](keyField: String): Write[K, Void, T] =
+    Write(keyField, None)
+
+  def write[K1: ClassTag, K2: ClassTag, T: ClassTag: Coder: ParquetType](
+    keyFieldPrimary: String,
+    keyFieldSecondary: String
+  ): Write[K1, K2, T] =
+    Write(keyFieldPrimary, Option(keyFieldSecondary))
```

Comment:
It should be safe to use `Some` here instead

Reply:
Yes, but more idiomatic to use `Option` and it'll save you if `null` is passed for some unknown reason.